campaignion_expiry/src/Anonymizer.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Drupal\campaignion_expiry;
+
+use Drupal\campaignion\Contact;
+
+/**
+ * Anonymization service for redhen contacts.
+ */
+class Anonymizer {
+
+  /**
+   * Anonymize a single contact.
+   *
+   * @param \Drupal\campaignion\Contact $contact
+   *   Contact that is being anonymized.
```

Comment:
The `Anonymizer` gets the contact passed as argument to the method call not to the constructor. That would also match the service definition.

```suggestion
```